documentation/docs/migration-guide.md

```diff
@@ -9,6 +9,74 @@ import { DemoWrapper } from '@site/docs/components/DemoWrapper';
+There are multiple reasons for this, but a major pain point has been that Reveal does not expose the three.js examples in its export, causing applications to have to install threejs anyway.
+In addition, we also see a lot of applications ending up having multiple bundles of three.js which we are trying to alliviate.
+
+## Exploded view tool has been removed
+Due to low / no usage of this tool we have decided to remove it.
+Note that the tool was a convenience method and it is fully possible to implement this in the application logic.
+The functionality should be easily adoptable from the removed tool.
+The removed tool can be found [here.](https://github.com/cognitedata/reveal/blob/release/3.3.x/viewer/packages/tools/src/ExplodedViewTool.ts)
+
+## Geomap tool has been removed
+Due to low / no usage of this tool we have decided to remove it.
+Note that the tool was a convenience method and it is fully possible to implement this in the application logic.
+The functionality should be easily adoptable from the removed tool.
+The removed tool can be found [here.](https://github.com/cognitedata/reveal/tree/release/3.3.x/viewer/packages/tools/src/Geomap)
+The geomap tool leveraged the [geo-three](https://www.npmjs.com/package/geo-three) npm library which can be used by your application and added through custom objects if you want to support this using Reveal.
+
+## Cognite3DViewer.setBackgroundColor now alpha
+The `setBackgroundColor` now supports settings alpha value. 
+This becomes relevant when you want your 3D content to blend with your website.
+
+## Cognite3DViewer.isBrowserSupported has been removed
+This method has been left unused for a long time, and Reveal now supports all major browsers with WebGL 2.0 capabilities.
+
+## Cognite3DModel has been renamed to CogniteCadModel
+Renamed for clarity, as technically a point cloud can also be viewer as a 3D model.
+
+## Cognite3DViewer.getIntersectionFromPixel options parameter has been removed
+Reveal currently uses GPU driven picking for point clouds, so the previously used options are not applicable anymore.
+
+## Cognite3DViewer.getCamera has been removed
+The camera must now be fetched from the `CameraManager`.
+The camera used by the current active `CameraManager` can be fetched with `viewer.cameraManager.getCamera()`.
+
+## Cognite3DViewer.renderer has been removed
+Getter for the renderer used by Reveal has been removed.
+
+## Cognite3DViewer.getScene has been removed
+Any direct manipulation of the three.js scene which Reveal uses for rendering can cause undefined behaviour, so in an effort to limit unintentional mistakes it has now been removed.
+Please reach out if there is any missing functionality that Reveal should expose.
+
+## CameraManager now requires two new methods: activate and deactivate
+These new methods are used for transitioning to and from other `CameraManager`s and lets each implementation decide what this functionality should look like.
```

Comment:
Perhaps we can move the latter section about `cameraStop` event handling in the CameraManager here as well? Just so that the CameraManager-related sections are bound together

Reply:
Makes sense, will fix!